### ISSUE TITLE
Bug: No metadata table at query time

### ISSUE THREAD

**Issue by taliaga** [smartsdk/ngsi-timeseries-api#115]
```
curl -X GET 'http://localhost:8668/v2/entities/Lamp:001/attrs/luminosity?=3&limit=3' 
-H 'Accept: application/json' -H 'Fiware-Service: openiot' -H 'Fiware-ServicePath: /'
"Something went wrong with QL. Error: SQLActionException[TableUnknownException: Table
 'doc.md_ets_metadata' unknown]"
```

This happened while running the FIWARE [step-by-step tutorial](https://fiware-tutorials.readthedocs.io/en/latest/time-series-data/index.html), which seems to have a container-startup-order-dependency issue (deploy, stop and start again to reproduce). Fix this as well.

**Comment by chicco785** [comment]
@c0c0n3 can you have a look if we can close it?

**Comment by c0c0n3** [comment]
@chicco785 yep, we can close it. tested the fix, it works. also @taliaga already added integration tests for that too. happy days!

**Comment by c0c0n3** [comment]
closed by #117
